.github/workflows/auto-assign.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Auto Assign'
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-reviews:
```

Comment:
I forgot to mention that `add-reviews:` now should ne `add-assignee:`. I can change it in my PR together with updating samples workflow.

Reply:
Yeah the name doesn't really matter, but you're right that it can be more precise - feel free to change it.